src/pybind/mgr/dashboard/module.py

```diff
@@ -417,6 +419,24 @@ def set_rgw_credentials(self):
 
         return 0, 'RGW credentials configured', ''
 
+    @CLIWriteCommand("dashboard set-login-banner-file")
```

Comment:
```suggestion
    @CLIWriteCommand("dashboard set-login-banner")
```
Actually you're not storing the filename but some data that coincidentally comes from a file, so no need to explicitly mention "file". The "-i <file>" is just a trick to feed a input parameter from a file, but we are not passing the file name, but the file contents.

Reply:
Ahh I see, have updated the code.